File: server/solarModel/matlab/dpp.m
function [ GHI ] = dpp( z,cosz )
%DPP model (1978)
DNI = 950.2*(1 - exp(-0.075*(90-abs(z) ) ) );
diff = 14.29 + 21.04 *( pi/2 - abs(z)*pi/180);
GHI = DNI*abs(cosz) + diff;
end

File: train/lib/info_riego_data/lib/solar_model/dpp.py
import math

def dpp(z, cosz):

    dni = 950.2 * (1 - math.exp(-0.075 * (90 - abs(z))))
    diff = 14.29 + 21.04 * (math.pi / 2.0 - abs(z) * math.pi / 180.0)
    ghi = dni * abs(cosz) + diff

    return ghi
